test/sharness/t0080-repo.sh

```diff
@@ -98,7 +104,7 @@ test_expect_success "remove recursive pin, add direct" '
 
 test_expect_success "remove direct pin" '
 	echo "unpinned $HASH" >expected6 &&
-	ipfs pin rm "$HASH" >actual6 &&
+	ipfs pin rm -r=false "$HASH" >actual6 &&
```

Comment:
Hrm... That was the behavior we initially intended to have. The idea was to mimic the unix `rm` command:

```sh
rm file #works
rm dir #doesnt work
rm -r file #works
rm -r dir #works
```

This does bring up the equivalent of having an `rmdir` analogue though.

What do you think @jbenet @Kubuxu @lgierth  ?